pkg/pipelines/service.go

```diff
@@ -195,3 +200,19 @@ func createSvcImageBinding(cfg *config.PipelinesConfig, env *config.Environment,
 	resourceFilePath := makeImageBindingPath(cfg, filename)
 	return name, filename, res.Resources{resourceFilePath: triggers.CreateImageRepoBinding(cfg.Name, name, imageRepo, strconv.FormatBool(isTLSVerify))}
 }
+
+func createConfigFolder(m *config.Manifest, appFs afero.Fs, o *AddServiceOptions) error {
+	basePath, err := homedir.Expand(o.PipelinesFolderPath)
+	if err != nil {
+		return fmt.Errorf("Cannot expand the pipelines.yaml path : %s", o.PipelinesFolderPath)
+	}
+	env := m.GetEnvironment(o.EnvName)
+	app := m.GetApplication(o.EnvName, o.AppName)
+	servicePath := config.PathForService(app, env, o.ServiceName)
+	finalPath := filepath.Join(basePath, servicePath, "base", "config")
+	err = appFs.MkdirAll(finalPath, 0755)
+	if err != nil {
```

Comment:
Will this fail if directory already exists?

Reply:
No William, it wont. if the directory is not present it will create one.

Reply:
https://golang.org/pkg/os/#MkdirAll